test/runtests.jl

```diff
@@ -31,8 +31,15 @@ end
     @test !Coverage.iscovfile("/otherdir/test.jl.cov", "/somedir/test.jl")
 end
 
-@testset "isfuncexpr" begin
```

Comment:
Did you mean to remove the test set here?

Reply:
Nope, fixed